automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/analysis/MethodLaunchSpec.scala

```diff
@@ -21,34 +22,6 @@ class MethodLaunchSpec extends FreeSpec /*with ParallelTestExecution*/ with Matc
   val missingInputsErrorText: String = "is missing definitions for these inputs:"
   val testData = TestData()
 
-  "launch workflow and delete a workflow" in {
```

Comment:
too many tests that launch workflow, causing long test execution time. `delete workflow` testing moved to `BillingSpec`